### PR TITLE
Portfolio mobile version

### PR DESCRIPTION
**Pull request by Isaac-ej**
This is the Portfolio project's finished mobile version Feature 
and was aimed at building the  section of the mobile website,
then this includes features as follows:
-work section, 
-about me section.
all the work is done with CSS flexbox and grid.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -49,10 +49,86 @@ <h2>LET'S CONNECT</h2>
...
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+            <div class="techs">
+              <ul>
+                <li>HTML</li>
+                <li> CSS</li>
+                <li>Javascript</li>
+              </ul>
+            </div>
+            <button>See Projects</button>
+          </div>
```
Comment (Cabraham1) [inline index.html]: - [ ] Great job on implementing the button! However, in order to align with the design requirements, please make the following updates:
- Apply the necessary CSS classes or styles to the button element to achieve the active, hover, and pressed button styles as specified in the design.

- [ ] Here is the required button style for each event
image

File: index.html [hunk 2]
```diff
@@ -49,10 +49,86 @@ <h2>LET'S CONNECT</h2>
...
+         </section>
```
Comment (Cabraham1) [inline index.html]: - [ ] Great progress on your work so far! 👍🏿 However, I've noticed that you have only created one card out of the four cards provided in the Figma template. According to the project requirements, I recommend that you create all the cards present in the Figma design for this particular project. It's important to ensure that all the necessary components are implemented. Keep up the good work! ✌🏿

File: index.html [hunk 3]
```diff
@@ -49,10 +49,86 @@ <h2>LET'S CONNECT</h2>
...
+                  </li>
+                  <li>
+                    <a href=""><img src="img/Vector.svg" alt=""></a>
+                  </li>
+                  <li>
+                    <a href="https://github.com/isaac-ej"><img src="img/Vector (1).svg" alt=""></a>
+                  </li>
+                </ul>
+                <button class="btn">Get my resume</button>
+
```
Comment (Cabraham1) [inline index.html]: - [ ] Great job on implementing the button! However, in order to align with the design requirements, please make the following updates:
- Apply the necessary CSS classes or styles to the button element to achieve the active, hover, and pressed button styles as specified in the design.

- [ ] Here is the required button style for each event
image

File: index.html [hunk 4]
```diff
@@ -49,10 +49,86 @@ <h2>LET'S CONNECT</h2>
...
+                  <li class="lis"><img src="https://upload.wikimedia.org/wikipedia/commons/3/3b/Javascript_Logo.png" alt="">Javascript</li>
+                  <li class="lis"><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/6/61/HTML5_logo_and_wordmark.svg/1200px-HTML5_logo_and_wordmark.svg.png" alt="">HTML</li>
+                  <li class="lis"><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/d/d5/CSS3_logo_and_wordmark.svg/1200px-CSS3_logo_and_wordmark.svg.png" alt="">CSS</li>
 
- 
+                <h3 class="under-lines">Frameworks<i class="fa-solid fa-chevron-right"></i></h3>
+                <h3 class="under-lines" >Skills<i class="fa-solid fas fa-chevron-right"></i></h3>
+                </div>
+              </section>
```
Comment (Cabraham1) [inline index.html]: - [ ] Great work on your progress! 👍🏿 However, I noticed that this section appears to be incomplete and some elements are missing from your design. I kindly suggest that you adhere to the Figma design and ensure that all the missing elements are added to complete this section. It's important to maintain consistency with the original design to ensure a cohesive and accurate representation. Keep up the good work and make sure to address these missing elements!

- [ ] Your Design

image

- [ ] Figma Design
image

**Review by Cabraham1 — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @Isaac-ej   :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:
- [ ] I kindly request that you adhere to the Figma design provided for your project. The Figma design serves as a visual reference and guideline for implementing the desired user interface and overall look of your application. Please ensure that the implemented code aligns with the design specifications provided in Figma. This includes aspects such as layout, color scheme, typography, and any other visual elements defined in the design.

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers, and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Isaac-ej) [reply index.html]: For your information, we were required to create only two sections:
work section and about section it was to choose only one

File: style.css [hunk 5]
```diff
@@ -113,11 +118,206 @@ h2 {
...
+  border: #6070ff solid 1px;
+  margin: 4rem 0;
+  transition: btn 3s;
+}
+
+button:hover,
+.btn:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
```
Comment (Kamba56) [inline style.css]: - [ ] Great job adding the hover state on your buttons. Kindly add the pressed state as suggested by the previous reviewer. You can use the `:active` pseudoclass for the pressed state.
image

File: index.html [hunk 6]
```diff
@@ -49,10 +49,193 @@ <h2>LET'S CONNECT</h2>
...
+            </div>
+            <button>See Projects</button>
+          </div>
+        </section>
+
+
+        
+         <section class="container-b">
+          <div class="bg-b">
+                <h1 class="about-h1">About Myself</h1>
```
Comment (Kamba56) [inline index.html:190]: - [Optional] Kindly note that it is best SEO practice to have just one h1 tag in your document. This is so screenreaders can identify the main title/heading of your document. Kindly change this to any other heading element `h2 --- h5`.

File: index.html [hunk 7]
```diff
@@ -49,10 +49,193 @@ <h2>LET'S CONNECT</h2>
...
+                <li>HTML</li>
+                <li> CSS</li>
+                <li>Javascript</li>
+              </ul>
+            </div>
+            <button>See Projects</button>
+          </div>
+        </section>
+
+        <section class="container">
```
Comment (Kamba56) [inline index.html:122]: - [Optional] This is what your work card looks like on the 375px screensize. Kindly fix the highlighted section to match the figma template.
### Your design
image
### Figma
image

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @Isaac-ej, 

Great work on making the changes requested by a previous reviewer 👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [x] GreaT JOB USING CSS GRID AND FLEXBOX TO layout contents!👏👏👏
- [x] 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 7 above)
Comment (rica213) [reply index.html:122]: - [ ]  [OPTIONAL] Kindly implement the required changes requested by the previous reviewer 🙏🏽 
Following the design from the Figma template as much as possible would demonstrate your attention to details skill, which is a **highly** sought-after skill by recruiters 👁️ 👁️

File: style.css (hunk 5 above)
Comment (rica213) [reply style.css]: - [ ] Please implement this change as both previous reviewers requested 🙏🏽 
You can achieve that this way 👇🏽 

```sh
button:active {
background-color: #2230D2;
color: #fff;
}
```

**Review by rica213 — CHANGES_REQUESTED** [review]
# 🟡 STATUS: REQUIRED CHANGES ♻️ 
Hi Maksim
My name is Larissa. Your code reviewer 🥰

Good job so far! 🤗 
I especially like your **commit messages**. Kudos 👍🏽 
However, I kindly ask you to implement all the required changes asked by previous reviewers so that we can approve your project 🙏🏽 
I **HIGHLY** recommend you implement **ALL** the **OPTIONAL** changes as well. Because they will enhance the quality of your code 💯 
Please do not hesitate to mention me in your comment if you have any questions ❓ ❔ 



Happy coding!
## Highlights
Good Design ✔️
Professional Documentation ✔️
No Linter errors ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by V-Blaze — CHANGES_REQUESTED** [review]
Hi @Isaac-ej  , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

dcentl8-9b96e129-9725-401b-ac34-2feb990202a0

## Invalid Code Review Request

## You have submitted a project, with an old pull request for a first-time review. I can see that it seems you are repeating the block, That is alright, You are required to create a new branch from this current branch, then create a new pull request, and then request for a review with the new pull request.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request closed without merging**